src/identity/index.ts

```diff
@@ -4,6 +4,7 @@ export { Avatar } from './components/Avatar';
 export { Badge } from './components/Badge';
 export { Name } from './components/Name';
 export { Identity } from './components/Identity';
+export { IdentityLayout } from './components/IdentityLayout';
```

Comment:
I feel like we should just offer one solution that combines Provider and Layout, and just keep it as `<Identity />`.

Having them split, can be done for internal purposes, but for external, I think we should keep just one. cc @kyhyco